README.md

```diff
@@ -208,6 +207,7 @@ POWER TOYS:
                            N=1: URL, N=2: URL and tag, N=3: title,
                            N=4: URL, title and tag
       -j, --json           Json formatted output for -p and search
+      --colors             --colors COLORS set output colors
```

Comment:
well googler's is:
--colors COLORS       set output colors (see man page for details)
do you what to use this one?